manifests/ml-training-on-eks/core-modules.yaml

```diff
@@ -0,0 +1,208 @@
+name: mwaa
+path: git::https://github.com/awslabs/idf-modules.git//modules/orchestration/mwaa?ref=release/1.2.0&depth=1
+dataFiles:
+  - filePath: data/mwaa/requirements/requirements-eks-operator.txt
+parameters:
+  - name: custom-requirements-path
+    value: data/mwaa/requirements/requirements-eks-operator.txt
+parameters:
+  - name: vpc-id
+    valueFrom:
+      moduleMetadata:
+        group: optionals
+        name: networking
+        key: VpcId
+  - name: private-subnet-ids
+    valueFrom:
+      moduleMetadata:
+        group: optionals
+        name: networking
+        key: PrivateSubnetIds
+  - name: dag-bucket-name
+    valueFrom:
+      moduleMetadata:
+        group: optionals
+        name: datalake-buckets
+        key: ArtifactsBucketName
+  - name: dag-path
+    value: dags
+  - name: airflow-version
+    value: "2.2.2"
```

Comment:
Not a fan of this version...but ok ;)

Reply:
MWAA is removed so this no longer applies